flink-runtime-web/web-dashboard/src/app/interfaces/safe-any.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type SafeAny = any;
```

Comment:
We should immediately mark this as deprecated. `SafeAny` is not safe at all, it's a hack that silences a correct warning/error. It would be better to use `unknown` instead everywhere, however that is a separate change as it isn't a drop-in replacement.